Here is the html I used:
```
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title></title>
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="1671.6">
  <style type="text/css">
    p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px 'Helvetica Neue'}
    p.p2 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px 'Helvetica Neue'; min-height: 14.0px}
    span.s1 {text-decoration: line-through}
  </style>
</head>
<body>
<p class="p1">Title</p>
<p class="p2"><br></p>
<p class="p1"><span class="s1">Strikethrough</span></p>
</body>
</html>
```

I'm just reading it in from a file and passing the contents to `unmark` and it crashes with the below error. This is because it tries to concatenate a DocType tag, which has no name.

```
Traceback (most recent call last):
  File "/Users/duncan/src/apple-notes-html-to-joplin/doctype-test.py", line 5, in <module>
    print(unmarkd.unmark(contents))
  File "/Users/duncan/src/apple-notes-html-to-joplin/unmarkd/unmarkd/__init__.py", line 11, in unmark
    return unmarkers.BasicUnmarker().unmark(html)  # type: ignore
  File "/Users/duncan/src/apple-notes-html-to-joplin/unmarkd/unmarkd/unmarkers.py", line 205, in unmark
    return self.__parse(html).strip().replace("\u0000", "\uFFFD")
  File "/Users/duncan/src/apple-notes-html-to-joplin/unmarkd/unmarkd/unmarkers.py", line 112, in __parse
    output += getattr(self, "tag_" + name)(child)
TypeError: can only concatenate str (not "NoneType") to str
```

--- a/tests/test_roundtrip.py
+++ b/tests/test_roundtrip.py
@@ -47,4 +47,5 @@ class TestExampleCases:
     def test_example_21(self) -> None: helper("```0")
     def test_example_22(self) -> None: helper("- Unordered lists, and:\n 1. One\n 2. Two\n 3. Three\n- More")
     def test_example_23(self) -> None: helper("a\n---\nb")
+    def test_example_24(self) -> None: assert unmarkd.unmark('''<!DOCTYPE html><html><body><p>A</p></body></html>''') == 'A'
 # fmt: on

